packages/react-charts/src/components/ChartBoxPlot/examples/ChartBoxPlot.md

```diff
@@ -0,0 +1,354 @@
+---
+id: Box plot chart
+section: charts
+propComponents: [
+  'Chart',
+  'ChartAxis',
+  'ChartBoxPlot',
+  'ChartCursorFlyout',
+  'ChartCursorTooltip',
+  'ChartVoronoiContainer'
+]
+hideDarkMode: true
+beta: true
+---
+
+import { Chart, ChartAxis, ChartBoxPlot, ChartBoxPlotTooltip, ChartCursorFlyout, ChartCursorTooltip, ChartLegendTooltip, ChartThemeColor, ChartThreshold, ChartVoronoiContainer, createContainer } from '@patternfly/react-charts';
+import { VictoryZoomContainer } from 'victory-zoom-container';
+import chart_color_orange_300 from '@patternfly/react-tokens/dist/esm/chart_color_orange_300';
+
+## Introduction
+Note: PatternFly React charts live in its own package at [@patternfly/react-charts](https://www.npmjs.com/package/@patternfly/react-charts)!
+
+PatternFly React charts are based on the [Victory](https://formidable.com/open-source/victory/docs/victory-chart/) chart library, along with additional functionality, custom components, and theming for PatternFly. This provides a collection of React based components you can use to build PatternFly patterns with consistent markup, styling, and behavior.
+
+
+## Examples
+### Basic with right aligned legend
+```js
+import React from 'react';
+import { Chart, ChartAxis, ChartBoxPlot } from '@patternfly/react-charts';
+
+<div style={{ height: '300px', width: '750px' }}>
+  <Chart
+    ariaDesc="Average number of pets"
+    ariaTitle="Bar chart example"
+    domain={{y: [0, 12]}}
+    domainPadding={{ x: [30, 25] }}
+    legendData={[{ name: 'Cats' }]}
+    legendOrientation="vertical"
+    legendPosition="right"
+    height={300}
+    name="chart3"
+    padding={{
+      bottom: 50,
+      left: 50,
+      right: 200, // Adjusted to accommodate legend
+      top: 50
+    }}
+    themeColor={ChartThemeColor.blue}
+    width={750}
+  >
+    <ChartAxis />
+    <ChartAxis dependentAxis showGrid />
+    <ChartBoxPlot
+      data={[
+        { name: 'Cats', x: '2015', y: [1, 2, 3, 5] },
+        { name: 'Cats', x: '2016', y: [3, 2, 8, 10] },
+        { name: 'Cats', x: '2017', y: [2, 8, 6, 5] },
+        { name: 'Cats', x: '2018', y: [1, 3, 2, 9] }
+      ]}
+    />
+  </Chart>
+</div>
+```
+
+### Green with bottom aligned legend
```

Comment:
Should the title be updated to match the `themeColor`?

Reply:
Yes. Thank you